packages/infrastructure/jest/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@k8slens/jest",
   "private": false,
-  "version": "0.0.1",
+  "version": "6.5.0-alpha.0",
```

Comment:
updates everything in @k8slens?

Reply:
That is what I put it too, maybe this is wrong... but I think it should be fine.